Configuration/StandardSequences/python/ReconstructionCosmics_cff.py

```diff
@@ -69,6 +69,7 @@
                                              jetsCosmics*
                                              muonsCosmics*
                                              regionalCosmicTracksSeq*
+                                             cosmicCDCTracksSeq*
```

Comment:
Are you suggesting to to remove the old regionalCosmicTracksSeq? and to call cosmicCDCTracksSeq: regionalCosmicTracksSeq?